main/test/ca/uwaterloo/flix/library/TestRead.flix

```diff
@@ -0,0 +1,40 @@
+/*
+ *  Copyright 2020 Matthew Lutze
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+namespace TestRead {
+    use Read.read;
```

Comment:
> Is it because when you declare a class its signatures become visible in that namespace?

Yes, the signature falls directly under the namespace of the class, not under a new namespace equivalent to the class. More info here: https://github.com/flix/flix/issues/1489#issuecomment-729656578